### ISSUE TITLE
Consider adding `composer drupal:run-server` to readme.md

### ISSUE THREAD

**Issue by isalmanhaider** [phenaproxima/starshot-prototype#45]
## Issue
After completing the installation of Drupal Starshot, launched in browser and initially logging in, I encountered a timeout issue. This problem can be particularly frustrating for users who are not familiar with how to restart the server.

## Suggestion
To enhance the usability and accessibility of the Drupal Starshot project for new and existing users, I recommend adding a section in the README.md file that includes essential commands for server management. Specifically, the inclusion of the `composer drupal:run-server` command would be very beneficial. 

## Justification
Adding this command can significantly aid users in managing their Drupal environment more effectively, especially in scenarios where the server times out or needs to be restarted for other reasons. It ensures that users have the necessary tools at their disposal to maintain system availability and stability.

## Additional Context
The following error prompted this suggestion:

```
The following exception is caused by a process timeout
Check https://getcomposer.org/doc/06-config.md#process-timeout for details

In Process.php line 1204:
                                                                                                  
The process "test -n "$CI" || composer drupal:run-server" exceeded the timeout of 300 seconds.
```

**Comment by phenaproxima** [comment]
Yeah, we've had others encounter the timeout issue, maddeningly. (pinging @goba)

The `run-server` command already invokes PHP with `-d max_execution_time=0`, in an effort to stop the timeout from happening, but I guess that hasn't been super effective.

Ideally we'd just fix the problem, rather than merely adding to documentation. That said, I don't think it's a bad idea to document this as a known issue. Can you file a PR to add this stuff to the README?

**Comment by phenaproxima** [comment]
Fixed by #46.
